src/evergreen/tst.py

```diff
@@ -13,12 +13,10 @@
 class Logs(_BaseEvergreenObject):
     """Representation of test logs from evergreen."""
 
-    url = evg_attrib("url")
     line_num = evg_attrib("line_num")
-    url_raw = evg_attrib("url_raw")
-    url_lobster = evg_attrib("url_lobster")
+    url = evg_attrib("url")
```

Comment:
both test and test log class fields have been reorganized to match the order in evergreen documentation [here](https://docs.devprod.prod.corp.mongodb.com/evergreen/API/REST-V2-Usage#tag/tests/paths/~1tasks~1%7Btask_id%7D~1tests/get).